src/services/esri-feature/types.d.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  esriGeometryPoint,
+  esriGeometryMultipoint,
+  esriGeometryPolyline,
+  esriGeometryPolygon,
+  esriGeometryEnvelope,
+} from '@esri/arcgis-rest-feature-layer';
+import type { IFeature } from '@esri/arcgis-rest-request';
+
+interface Feat extends IFeature {
+  attributes: {
+    aoiId: string;
+    area: number;
+    areaName: string;
+    elu: string;
+    isCustom: boolean;
+    per_global: number;
+    percentage: number;
+    population: number;
+    pressures: string;
+    protectedAreasList: string;
+    species: string;
+    speciesNumbers: {
+      nspecies: number;
+    };
+    time_stamp: number;
+  };
+}
+
+export interface AddFeatureProps {
+  features: Feat;
+  url: string;
+}
+
+export interface GetFeauturesProps {
+  geometry?:
```

Comment:
Dont we have this already from the IFeature type?

Reply:
But maybe we can extract the geometry from IFeature

Reply:
ok, done.